libs/wingsdk/src/shared-tf/app.ts

```diff
@@ -34,7 +34,7 @@ export abstract class CdktfApp extends App {
   public readonly terraformManifestPath: string;
   public readonly outdir: string;
   public readonly isTestEnvironment: boolean;
-  public readonly _tokens: Tokens;
```

Comment:
What's the motivation to keep this field if tokens are now global?

Reply:
There is none, just haven't reworked the tokens to make sense in that context